backend/api/v1/database_service.go

```diff
@@ -127,14 +127,15 @@ func (s *DatabaseService) SearchDatabases(ctx context.Context, request *v1pb.Sea
 		return nil, err
 	}
 
-	permission := iam.PermissionDatabasesGet
+	permission := iam.PermissionDatabasesList
```

Comment:
Looks like a mistake? We should need databases.list permission instead of databases.get for database search @d-bytebase

Reply:
Search database does use Get() permission.

https://cloud.google.com/resource-manager/reference/rest/v3/projects/search
Search for projects that the caller has the resourcemanager.projects.get permission on, and also satisfy the specified query.

Reply:
Revert this change